### PR TITLE
feat(demo): reduce bg image size

### LINKED ISSUE #310: [FEATURE] reduce background-size images
### Description

Reduce background images displayed on CMS pages.

### Use Case

Go to: https://frontends-demo.vercel.app/Summer-BBQ/
 you can see that background image has 2MB of size and it's not cropped like other images
image

### Proposed Solution

1. At least we should attach image helper to reduce image size, for example `?width=800&fit=crop,smart` - dynamically like in other images
2. The best would be also to have a solution like `load="lazy"`. It does not exist on CSS, so maybe moving that to separate `img` tag or similar magic would do the trick

### Alternatives Considered

Reducing these images size and loading them on demand will improve performance of the CMS pages

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by BrocksiNet**
### Description

closes #310

### Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### Screenshots (if applicable)

Image sizes before changes (prod):
image

Image sizes after changes (dev):
image

### Additional context
Added a new helper to change the background image URL.

### CODE REVIEW

**Review by kstala — APPROVED** [review]

File: packages/helpers/src/cms/getBackgroundImageUrl.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  return num ? Math.ceil(num / 100) * 100 : DEFAULT_BG_IMAGE_SIZE;
+}
+
+const getUrlFromBackgroundImage = (url: string) => {
+  const regex = /(?:\(['"]?)(.*?)(?:['"]?\))/;
+  const match = url.match(regex);
+  if (!match) {
+    return url;
+  }
+  return match[1];
```
Comment (mdanilowicz) [inline packages/helpers/src/cms/getBackgroundImageUrl.ts:14]:
```suggestion
return !match ? url : match[1]
```

File: packages/helpers/src/cms/getBackgroundImageUrl.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { CmsBlock, CmsSection } from "@shopware-pwa/types";
+
+const DEFAULT_BG_IMAGE_SIZE = 800;
+function roundUp(num: number) {
+  return num ? Math.ceil(num / 100) * 100 : DEFAULT_BG_IMAGE_SIZE;
+}
+
+const getUrlFromBackgroundImage = (url: string) => {
+  const regex = /(?:\(['"]?)(.*?)(?:['"]?\))/;
```
Comment (mdanilowicz) [inline packages/helpers/src/cms/getBackgroundImageUrl.ts:9]: We should change the variable name or add a comment to describe this regex

**Review by mdanilowicz — APPROVED** [review]

File: packages/helpers/src/cms/getBackgroundImageUrl.ts (hunk 2 above)
Comment (BrocksiNet) [reply packages/helpers/src/cms/getBackgroundImageUrl.ts:9]: What would be a better variable name?

Comment (BrocksiNet) [reply packages/helpers/src/cms/getBackgroundImageUrl.ts:9]: I added tests for the new helper so I guess it is clear what this regex is doing now.

**Pull request merged**